src/components/json-editor.tsx

```diff
@@ -22,13 +23,16 @@ type Props = {
 };
 
 export default function JsonEditor({
-  initial = '{\n  "hello": "world"\n}',
+  initial,
```

Comment:
было лучше, с 


```suggestion
  initial = '{\n  "hello": "world"\n}',
```

только '{\n  "hello": "world"\n}' бы еще в константу вытащить